src/metrics/listener.ts

```diff
@@ -56,7 +54,7 @@ export interface MetricsConfig {
 }
 
 export class MetricsListener {
-  private currentProcessor?: Promise<Processor>;
+  private currentProcessor?: Promise<any>;
```

Comment:
Sadly, this has to stay as it, so we don't import and load the modules that add 1MB of memory allocation.